Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -302,8 +302,21 @@ WarpXParticleContainer::AddNParticles (int /*lev*/, long n,
 #ifdef AMREX_USE_EB
     auto & distance_to_eb = WarpX::GetInstance().GetDistanceToEB();
     scrapeParticles( *this, amrex::GetVecOfConstPtrs(distance_to_eb), ParticleBoundaryProcess::Absorb());
-    // Call (local) redistribute again to remove particles with invalid ids
-    Redistribute(0, -1, 0, 1, true);
+    deleteInvalidParticles();
```

Comment:
The `#endif` currently on line 320 should be directly below this line. Currently the `WarpXParticleContainer::deleteInvalidParticles` function is only defined for EB=ON compilations.

Reply:
Thanks! Done.